python/GafferSceneUI/LightEditor.py

```diff
@@ -339,12 +366,24 @@ def __editSelectedCells( self, pathListing, quickBoolean = True ) :
 		inspectors = {}
 		inspections = []
 
+		with Gaffer.Context( self.getContext() ) as context :
```

Comment:
Do we need to scope a separate context here? or could `lightSetMembers =...` move down to below L372?

Reply:
Nope, no need for the double scope. Fixed in https://github.com/GafferHQ/gaffer/pull/5895/commits/56bb1983cfcf80f98666cad9d9b7ec0fe15ed1e3